examples/hello-world/step-by-step/cifar10/readme.md

```diff
@@ -0,0 +1,18 @@
+
+# Training a image classifier with CIFAR10 data
+ 
+We will use the original [Training a Classifer](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html) example
+in pytorch as the code base. In the follow examples, we will show various Federated Learning workflows. 
```

Comment:
"In the follow examples, we ..." should be below the image and description of cifar10.